### ISSUE TITLE
sniffer

### ISSUE THREAD

**Issue by JohanLeirnes** [kobuki/RFM69OOK#1]
Hello,

Could your OOKDumper be used to sniff out what signal a remote is using to send a signal that the outlets are reacting to and then using that data to replicate the signal?

Im trying but when i open serialmonitor i only get "start" and then nothing.

**Comment by kobuki** [comment]
Yeah, you can do that but it's a very primitive tool (mostly just a POC). I've done a few recordings and have succesfully decoded a simple weather transmitter but there are far better tools for this purpose.

OTOH, you should make sure you use the right band - 433/868/915 MHz. What kind of transmitter do you have?

**Comment by JohanLeirnes** [comment]
I have a Everflourish EMW100T transmitter. (and ofc a couple of EMW100R recievers)
I have a 433mhz RFM69HW and the defined frequency is 433.9 MHz in the OOKDumper.

I actually have no idea if that transmitter is using OOK but becuase most do I thought a would give it a try.

There seem to be some cheksum defenitions in play with this transmitter. 
Found some info about it here (including someone who has cracked the code but as im not very handy with C I have no idea on how to implement his code to a working arduino sketch. 

https://forum.telldus.com/viewtopic.php?f=2&t=97&start=60#p3335
(excuse the swedish)

Also here is someone who has somehow read the signal, no idea how :)

Beginning+ID+IDEND+Command

Beginning: 0-0-0-0-0-0--0--0-0-0--0--0-0-0--0-0--0--0-0--0-0-0--0-0--0--0-0-0--0--0-0-0--0-
80 bits

Id for 1: 0--0-0--0--0-0-0--0--0-0-0-
Id for 2: 0--0--0-0-0--0--0-0-0--0--0
Id for 3: -0-0-0--0-0--0-0--0-0--0-0-
Id for 4: -0-0--0-0--0-0-0--0-0--0-0-
27 bittiä

IDEND: -0--0-0-
8 bits

Command off: -0-0--0-0--0-0
Command  on: 0--0--0-0-0--0
14 bits

Message: 129bits
Transmission time: ~80ms

What other tools do you recommend on doing this, ofc the easy way is to buy a tellstick and just use that but whats the challange in that :P

**Comment by kobuki** [comment]
Sorry, I was quite busy last week. TBH, it was a long time ago when I looked for a similar thing but I encountered a few. I'd suggest having a look at JeeLab's repository, there's an OOK implementation for the RFM12B, IIRC with scanners and decoder tools. It would be easy to adapt, I think. There's also a Windows app called AnalysIR, for IR codes but last time I looked it seemed like something that could be used for OOK analysis too. It's not free though but has a working demo. I'm closing this now, if you have any comments or questions just re-open the ticket.
